beanie/odm/documents.py

```diff
@@ -689,17 +688,17 @@ async def update(
         self,
         *args,
         ignore_revision: bool = False,
-        session: Optional[ClientSession] = None,
+        session: Optional[AsyncIOMotorClientSession] = None,
         bulk_writer: Optional[BulkWriter] = None,
         skip_actions: Optional[List[Union[ActionDirections, str]]] = None,
         skip_sync: Optional[bool] = None,
-        **pymongo_kwargs,
-    ) -> DocType:
+        **pymongo_kwargs: Any,
+    ):
```

Comment:
Why you remove return type? 
If you using mypy as a type checker you will facing Error

Reply:
If it doesn't return any thing 
You can using '-> None'